packages/serve-runtime/src/useCompleteSubscriptionsOnDispose.ts

```diff
@@ -0,0 +1,51 @@
+import { createGraphQLError, isAsyncIterable, Repeater } from 'graphql-yoga';
+import type { MeshServePlugin } from './types';
+
+export function useCompleteSubscriptionsOnDispose(
+  disposableStack: AsyncDisposableStack,
+): MeshServePlugin {
+  function createShutdownError() {
+    return createGraphQLError('subscription has been closed because the server is shutting down', {
+      extensions: {
+        code: 'SHUTTING_DOWN',
+      },
+    });
+  }
```

Comment:
We can avoid re-creating the error. It'll never change.

```suggestion
const shutdownError = createGraphQLError('subscription has been closed because the server is shutting down', {
  extensions: {
    code: 'SHUTTING_DOWN',
  },
})
```